packages/node-opentelemetry/src/otel.ts

```diff
@@ -18,11 +19,24 @@ const LOG_PREFIX = `⚠️  ${_LOG_PREFIX}`;
 
 const env = process.env;
 
+const customInstrumentationMap = {
+  '@opentelemetry/instrumentation-remix': RemixInstrumentation,
+};
+
+type ConfigArg<T> = T extends new (...args: infer U) => unknown ? U[0] : never;
+export type CustomInstrumentationConfigMap = {
+  [Name in keyof typeof customInstrumentationMap]?: ConfigArg<
+    (typeof customInstrumentationMap)[Name]
+  >;
+};
+
 type SDKConfig = {
   advancedNetworkCapture?: boolean;
   betaMode?: boolean;
   consoleCapture?: boolean;
   instrumentations?: InstrumentationConfigMap;
+  additionalInstrumentations?: CustomInstrumentationConfigMap[];
```

Comment:
we should allow users to pass any 3rd party custom instrumentation pkgs. Can check out the type (https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation/src/types_internal.ts#L21)

Reply:
Gotcha, switched over to InstrumentationOption!